Code/GraphMol/MolAlign/testMolAlign.cpp

```diff
@@ -384,8 +398,9 @@ void testMMFFO3AConstraints() {
   TEST_ASSERT(o3a);
   o3a->align();
   delete o3a;
-  double d = (m2.getConformer().getAtomPos(cIdx) -
-              m1->getConformer().getAtomPos(cIdx)).length();
+  double d =
+      (m2.getConformer().getAtomPos(cIdx) - m1->getConformer().getAtomPos(cIdx))
```

Comment:
This is what I use:
```
clang-format version 3.9.0 (tags/RELEASE_390/final)
```
it uses the .clang-format file from RDkit, I checked this.

Could be a change in the formatter. I've seen this in yapf as well.